### DISCUSSION TITLE
LDAP Connection Type how to create?

### DISCUSSION THREAD

**Discussion by AlexeyMishenkov** [apache/airflow#30019] [Q&A]
Hello!

I'd like to execute PowerShell script in Airflow and install apache-airflow-providers-microsoft-psrp. As I understand I should create Connection with Connection Type - LDAP, but where are no LDAP in drop down list. How create Connection properly?

Providers instaled:
pip install apache-airflow-providers-microsoft-psrp==2.2.0

![image](https://user-images.githubusercontent.com/117362528/224295877-a8eae1ad-54d2-4584-8631-98d03c615f8d.png)

**Comment by potiuk** [comment]
You can use generic connection type. I don't think PSRP has specific connection type - looking at the code you can simply define extras that correspond to WSMan parameters - but maybe @malthe might help here - he was doing most of the PSRP improvements at some point in time,

**Reply by malthe** [reply]
Yes, any connection can be used, but the extras might be interesting, for example to specify a [configuration name](https://github.com/jborean93/pypsrp/blob/bfea572c29fcf8aac9c0625e7f7bb4ee4002d50e/src/pypsrp/powershell.py#L102).

That works with any connection type though.
